gdbgui/static/js/gdbgui.js

```diff
@@ -695,6 +695,8 @@ const GdbConsoleComponent = {
         GdbCommandInput.set_input_text(previous_cmd_from_history)
         // put focus back in input so user can just hit enter
         GdbCommandInput.el.focus()
+        //reset up-down arrow cmd history index
+        localStorage.setItem("cache-cmd-index", 0);
```

Comment:
Single quotes everywhere `'` unless double quotes are required. Also, another convention in the file is no semicolons.